project_status/__init__.py

```
@@ -0,0 +1 @@
+from . import models
```

Ruff (0.11.9)

`.models` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)